src/index.html

```diff
@@ -1,13 +1,381 @@
-<!DOCTYPE html>
-<html lang="en">
+<!doctype html>
+<html
+  class="page"
+  lang="en"
+>
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
-    <script src="scripts/main.js" defer></script>
+    <meta
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      name="viewport"
+    >
+    <meta
+      content="ie=edge"
+      http-equiv="X-UA-Compatible"
+    >
+    <link
+      href="https://fonts.googleapis.com/css?family=Poppins"
+      rel="stylesheet"
+    >
+    <link
+      href="./styles/style.scss"
+      rel="stylesheet"
+    >
+    <title>MyBike</title>
+    <link
+      href="icons/favicon.svg"
+      rel="shortcut icon"
+      type="image/x-icon"
+    >
+    <script
+      defer
+      src="scripts/main.js"
+    ></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <body class="page_body">
+    <header class="header">
+      <div class="header_top">
+        <a
+          class="menu_logo"
+          href="#"
+        >
+          <img
+            alt="Bike Logo"
+            class="logo"
+            src="images/logo.png"
+          >
+        </a>
+        <div class="header_icons">
+          <a
+            class="icon icon_phone"
+            href="tel:+1 234 5555-55-55"
+          ></a>
+          <a
+            class="icon icon_menu"
+            href="#menu"
+          ></a>
+        </div>
+      </div>
+      <div class="header_bottom">
+        <h1 class="header_title">Take the Streets</h1>
+      </div>
+    </header>
+
+    <aside
+      class="page_menu menu"
+      id="menu"
+    >
+      <div class="header_top menu_top">
+        <a
+          class="menu_logo"
+          href="#"
+        >
+          <img
+            alt="Bike Logo"
+            class="logo"
+            src="images/logo__menu.png"
+          >
+        </a>
+        <a
+          class="icon icon_close"
+          href="#"
+        ></a>
+      </div>
+
+      <div class="menu_content">
+        <nav class="nav">
+          <ul class="nav_list ">
+            <li class="nav_item">
+              <a
+                class="nav_link"
+                href="#"
+              >Home</a>
+            </li>
+            <li class="nav_item">
+              <a
+                class="nav_link"
+                href="#about_us"
+              >About us</a>
+            </li>
+            <li class="nav_item">
+              <a
+                class="nav_link"
+                href="#compare_bikes"
+              >Compare Bikes</a>
```

Comment:
Need to fix code style everywhere
```suggestion
              <a
                class="nav_link"
                href="#compare_bikes"
              >
                Compare Bikes
              </a>
```